### ISSUE TITLE
kinda buggy but very funny

### ISSUE THREAD

**Issue by jaylac2000** [ir8scripts/ir8-snatch#1]
when i try to snatch someone it sometimes bugs out and the peds cant be killed

also suggestion to lock vehicle doors so people cant leave

**Comment by RobertGrubb** [comment]
The peds should be set to be invincible by default from the config.

If you could do me a favor and set that to false and let me know how that goes, if it fails to change I'll take another look.

I will also look at the vehicle doors being locked, oddly I thought I threw that in but I have to double check.

Thanks for the input!

**Comment by jaylac2000** [comment]
invincible is fixed! i noticed it in the config after. it was becoming a problem where peds would get bugged out and anyone who bumps into them gets killed and the peds cannot be killed. luckily most of this was admin trolling against each other but good script and concept though. just doors dont lock so player can leave. sometimes the peds get stuck on fences either on foot oor vehicle but sometimes thats just GTA AI traffic. maybe add a check if they set stuck for so long it teleports them into the vehicle, and if the vehicle gets stuck for so long it will just cancel the action

**Comment by jaylac2000** [comment]
had a rea; good rp scene though where 'some corrupt police, just took someone'

**Comment by RobertGrubb** [comment]
@jaylac2000 

So in terms of the original piece for opening the ticket with NPCs being invincible, we worked that out via config variables.

The rest of the suggestions in the ticket, I've jotted down and will be making it into a future release (Give me a week or so, I usually get my dev time for FiveM on the weekends). 

With that said, I'm going to close this ticket, but stay on the lookout for future updates.

Thanks again for your suggestions!
